google/cloud/pubsub_v1/gapic/subscriber_client.py

```diff
@@ -1493,7 +1499,13 @@ def update_snapshot(
             >>>
             >>> seconds = 123456
             >>> expire_time = {'seconds': seconds}
-            >>> snapshot = {'expire_time': expire_time}
+            >>> snapshot_name = 'projects/my-project/snapshots/my-snapshot'
+            >>> topic_name = 'projects/my-project/topics/my-topic'
+            >>> snapshot = {
+            ...    'name': snapshot_name,
+            ...    'topic': topic_name,
```

Comment:
As per `.proto` file, neither `name` nor `topic` are required, but I believe at least the `name` field should be, thus added both fields to the sample, just in case.

Reply:
We should remove "topic" for the same reason as above.